streaming/python/runtime/task.py

```diff
@@ -83,9 +83,11 @@ def exit_handler():
             import atexit
             atexit.register(exit_handler)
 
+        # TODO(chaokunyang) add task/job config
         runtime_context = RuntimeContextImpl(
             self.worker.execution_task.task_id,
-            self.worker.execution_task.task_index, execution_node.parallelism)
+            self.worker.execution_task.task_index, execution_node.parallelism,
```

Comment:
```
a,b,c,d 
```
or
```
a,
b,
c,
d
```
for better readable?

Reply:
Since `a,b,c,d` here will be exceed line width `79`, so I put it in next line

Reply:
It's also enforced by python lint